bin/critterget.py

```diff
@@ -632,40 +676,42 @@ def main():
         else:
             continue
 
-        getDailyAppLoads(key,apps[key])
-        getDailyCrashes(key,apps[key])
-        getCrashCounts(key,apps[key])
-
-        getGenericPerfMgmt(key,apps[key],"volume","device","DailyVolumeByDevice")
-        getGenericPerfMgmt(key,apps[key],"errors","service","DailyServiceErrorRates")
-        getGenericPerfMgmt(key,apps[key],"volume","os","DailyVolumeByOS")
-        getGenericPerfMgmt(key,apps[key],"volume","appVersion","VolumeByAppVersion")
-
-        getGenericErrorMon(key,apps[key],"crashes","device","CrashesByDevice")
-        getGenericErrorMon(key,apps[key],"crashPercent","device","CrashPerByDevice")
-        getGenericErrorMon(key,apps[key],"appLoads","os","ApploadsByOs")
-        getGenericErrorMon(key,apps[key],"crashes","os","DailyCrashesByOs")
-        getGenericErrorMon(key,apps[key],"crashPercent","os","CrashPerByOs")
-        getGenericErrorMon(key,apps[key],"crashPercent","appVersion","CrashPerByAppVersion")
-        getGenericErrorMon(key,apps[key],"crashes","appVersion","CrashByAppVersion")
-        getGenericErrorMon(key,apps[key],"appLoads","appVersion","LoadsByAppVersion")
-        getGenericErrorMon(key,apps[key],"dau","appVersion","DauByAppVersion")
-        getGenericErrorMon(key,apps[key],"appLoads","device","ApploadsByDevice")
-
-        getAPMEndpoints(key, apps[key], LATENCY, "ApmEndpointsLatency")
-        getAPMEndpoints(key, apps[key], VOLUME, "ApmEndpointsVolume")
-        getAPMEndpoints(key, apps[key], ERRORS, "ApmEndpointsErrors")
-        getAPMEndpoints(key, apps[key], DATA, "ApmEndpointsData")
-
-        getAPMServices(key, apps[key], LATENCY, "ApmServicesLatency")
-        getAPMServices(key, apps[key], VOLUME, "ApmServicesVolume")
-        getAPMServices(key, apps[key], ERRORS, "ApmServicesErrors")
-        getAPMServices(key, apps[key], DATA, "ApmServicesData")
-
-        getAPMGeo(key, apps[key], LATENCY, "ApmGeoLatency")
-        getAPMGeo(key, apps[key], VOLUME, "ApmGeoVolume")
-        getAPMGeo(key, apps[key], ERRORS, "ApmGeoErrors")
-        getAPMGeo(key, apps[key], DATA, "ApmGeoData")
+        getTrends(key,apps[key]['name'])
+
+        getDailyAppLoads(key,apps[key]['name'])
+        getDailyCrashes(key,apps[key]['name'])
+        getCrashCounts(key,apps[key]['name'])
+
+        getGenericPerfMgmt(key,apps[key]['name'],"volume","device","DailyVolumeByDevice")
```

Comment:
We're using a mix of quotes here and below.  Let's use single quotes where possible please.